main.js

```diff
@@ -37,6 +39,30 @@ if (pathToParamedicConfig || // --config
         ParamedicConfig.parseFromFile(pathToParamedicConfig):
         ParamedicConfig.parseFromArguments(argv);
 
+    if(argv.plugin) {
+        paramedicConfig.setPlugins(argv.plugin);
+    }
+
+    if(argv.outputDir) {
```

Comment:
Do you assume the outputDir exists? Perhaps you validate somewhere else - should you instead validate here?